samples/java-protobuf-tracing/.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+	"java.configuration.updateBuildConfiguration": "interactive"
```

Comment:
.gitignore?

Reply:
ups

Reply:
removed. I'm not actually using the VS anymore